examples/document-brochure.html

```diff
@@ -0,0 +1,268 @@
+{% extends "_layouts/site.html" %}
```

Comment:
This file got renamed from `document.html`, it's contents didn't change.

Doesn't need a review on its own.